### PR TITLE
Migrate to python3 + cleanup

### LINKED ISSUE #282: Switch to Python 3

### PR DESCRIPTION
**Pull request by calixteman**
Fixes #282.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-sudo: required
+dist: xenial
```
Comment (marco-c) [inline .travis.yml]: Why is this needed?

File: auto_nag/scripts/missing_beta_status.py [hunk 2]
```diff
@@ -64,13 +64,10 @@ def handle_bug(self, bug, data):
...
 
     def get_bz_params(self, date):
-        self.status_central = utils.get_flag(
-            self.versions['central'], 'status', 'central'
-        )
-        self.status_release = utils.get_flag(
-            self.versions['release'], 'status', 'release'
-        )
-        self.status_beta = utils.get_flag(self.versions['beta'], 'status', 'beta')
+        r, b, c, _ = utils.get_versions()
```
Comment (marco-c) [inline auto_nag/scripts/missing_beta_status.py]: Nit: use full names here, one letter is confusing

File: auto_nag/utils.py [hunk 3]
```diff
@@ -471,3 +471,48 @@ def get_human_lag(date):
...
     return humanize.naturaldelta(today - dt)
+
+
+def get_version(jsonVersion, key):
+    # In X.Y, we just need X
+    version = jsonVersion[key].split('.')
+    return version[0]
+
+
+def get_current_versions():
```
Comment (marco-c) [inline auto_nag/utils.py]: Nit: you could use libmozdata here

File: runauto_nag_daily.sh [hunk 4]
```diff
@@ -4,5 +4,114 @@ set -e
 export PYTHONPATH=.
```
Comment (marco-c) [inline runauto_nag_daily.sh]: Double-check you didn't miss anything when moving from the now deleted runauto_nag_... files

Comment (marco-c) [reply runauto_nag_daily.sh]: Only `regression_but_type_enhancement_task` seems to be missing, in the hourly runner script.

File: .travis.yml (hunk 1 above)
Comment (calixteman) [reply .travis.yml]: because of python 3.7:
https://docs.travis-ci.com/user/languages/python/#python-37-and-higher

File: auto_nag/utils.py (hunk 3 above)
Comment (calixteman) [reply auto_nag/utils.py]: Yes you're right: I almost just moved the code.
So let's do that in an other commit

**Comment by sylvestre** [comment]
```
      'f3': utils.get_flag(self.versions['release'], 'tracking', 'release'),
   TypeError: 'NoneType' object is not subscriptable
```

**Comment by sylvestre** [comment]
please fix that

**Comment by marco-c** [comment]
The error is in auto_nag/scripts/tracked_bad_severity.py:
```
   Traceback (most recent call last):
    auto_nag/tests/test_bzcleaner.py line 51 in test_get_bz_params
      p = TrackedBadSeverity().get_bz_params(None)
    auto_nag/scripts/tracked_bad_severity.py line 31 in get_bz_params
      'f3': utils.get_flag(self.versions['release'], 'tracking', 'release'),
```

**Comment by calixteman** [comment]
The bug is because of non-following versions in p-d when the test ran yesterday. I'm fixing it.
